src/modules/sdk/objections/service.ts

```diff
@@ -57,13 +57,18 @@ export const getCompanyEligibility = async (companyNumber: string, token: string
  * @returns {string} the id of the newly created objection
  * @throws {ApiError}
  */
-export const createNewObjection = async (companyNumber: string, token: string, createWithData: ObjectionCreate): Promise<string> => {
+export const createNewObjection = async (companyNumber: string, token: string, createWithData: ObjectionCreate): Promise<ObjectionCreatedResponse> => {
```

Comment:
api change means it returns id and status so I've created an ObjectionCreatedResponse type to capture these. It used to just return the id as a string.